ui/entry-perf.js

```diff
@@ -27,11 +27,11 @@ import './js/filters';
 import './js/models/perf/issue_tracker';
 import './js/models/perf/performance_framework';
 import './js/models/perf/alerts';
-import './js/services/perf/math';
-import './js/services/perf/compare';
 import './js/controllers/perf/compare';
 import './js/controllers/perf/graphs';
 import './js/controllers/perf/alerts';
 import './js/components/perf/compare';
 import './js/components/loading';
 import './js/perfapp';
+import './perfherder/constants';
+import './perfherder/helpers';
```

Comment:
These files are imported at point of use (since they don't use the legacy angular module system), so don't need to be added here.

Reply:
yeah, I meant to remove those - will do so.